module/PowerShellEditorServices/Commands/Public/Clear-Host.ps1

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+Microsoft.PowerShell.Management\Get-Item function:Clear-Host | Microsoft.PowerShell.Management\Set-Item function:__clearhost
+
+function Clear-Host {
+    [Alias('cls')]
+    param(
+        [Parameter()]
+        [switch]
+        $All
+    )
+
+    __clearhost
+    if ($All.IsPresent) {
```

Comment:
Is there a good reason to include `.IsPresent`?

Reply:
I think strict mode requires this.